### PR TITLE
[bug 1127428] Uncomment booleanfield test

### PR DESCRIPTION
**Pull request by willkg**
The BooleanField required=True problem is gone now. I uncommented out
the test, but then bumped into a goofy edge case where BooleanField
defaults to False if you do an HTML form post.

So I went through all the test cases and converted them all to JSON
payloads. That fixes this test and is more correct test-wise since our
API docs ask people to use JSON payloads.

Quick r?

### CODE REVIEW

File: fjord/feedback/tests/test_api.py [hunk 1]
```diff
@@ -580,26 +605,28 @@ def test_invalid_email_address_returns_400(self):
...
+            'product': u'Firefox OS',
+            'locale': 'en-US',
+        }
+
+        r = self.client.post(
+            reverse('feedback-api'),
+            content_type='application/json',
+            data=json.dumps(data))
+        eq_(r.status_code, 400)
+        assert 'happy' in r.content
```
Comment (willkg) [inline fjord/feedback/tests/test_api.py]: This is the test we uncommented. Works super now.

**Comment by mythmon** [comment]
I read the uncommented test and skimmed over the rest. Looks good.

r+

**Comment by willkg** [comment]
Landed in 47db8d6 [bug 1127428] Uncomment booleanfield test

Thank you!
